README.md

```diff
@@ -1,2 +1,24 @@
-# ms-office-macro-nuke
-Tool to Purge and Remove all Macro And Dynamic Content from MS Office File
+![MS Office Macro Bleach](docs/images/bleach.jpg)
+
+
+# MS Office Macro Bleach
+
+A command-line tool designed to detect and purge any and all macros and dynamic content from MS Office files.
+
+It currently supports '.docm' and'.docx' files and will continue to expand its file support with every released version. The next file types to be supported will be '.pdf'' and '.doc'.
+
```

Comment:
I think you need to look broader into this topic. For example I believe the current format you added support for, is used by other Ms Products. I did some quick research with ChatGPT and this is what I found. 

The format used by Office 2010 files that resembles a zip format is called Office Open XML (OOXML). 

Microsoft Word
.docx: Word Document
.docm: Word Document with Macros
.dotx: Word Template
.dotm: Word Template with Macros

Microsoft Excel
.xlsx: Excel Spreadsheet
.xlsm: Excel Spreadsheet with Macros
.xltx: Excel Template
.xltm: Excel Template with Macros

Microsoft PowerPoint
.pptx: PowerPoint Presentation
.pptm: PowerPoint Presentation with Macros
.potx: PowerPoint Template
.potm: PowerPoint Template with Macros
.ppsx: PowerPoint Show
.ppsm: PowerPoint Show with Macros

Microsoft Visio
.vsdx: Visio Drawing
.vsdm: Visio Drawing with Macros
.vstx: Visio Template
.vstm: Visio Template with Macros

Microsoft Project
.mppx: Project Plan (Note: while Project files are often just .mpp, newer versions can save as .mppx in line with other Office products

Maybe you can generate more files with a Macro and save it on this extensions, add it to the test files, and confirm if you can claim already support for this formats.